src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        int value = (int) (1 + Math.random() * 10);
```

Comment:
10 is a magic number
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names